cfme/services/catalogs/catalog_items/__init__.py

```diff
@@ -245,10 +245,11 @@ class BaseCatalogItem(BaseEntity, Updateable, Pretty, Taggable):
 
     @property
     def button_icon_name(self):
-        return 'Button Image 1' if self.appliance.version < '5.9' else 'broom'
+        return 'broom'
 
     def update(self, updates):
         view = navigate_to(self, 'Edit')
+        view.wait_displayed(timeout="5s")
```

Comment:
What's happening here? view.wait_displayed is called by default on navigation, there should be no need to call this again? Are you observing slow page load, is_displayed returning `True` when the page isn't ready, some specific widget isn't loading quickly enough?